### ISSUE TITLE
libpulsecommon-16.1.dylib not found at application launch

### ISSUE THREAD

**Issue by jdykstra** [jdykstra/quodlibet#2]
Quod Libet pre-alpha.  This is visible when the app is launched from the command line:

```
(gst-plugin-scanner:80043): GStreamer-WARNING **: 15:23:12.867: Failed to load plugin '/Users/jwd/nobackup-cs/quodlibet/dev-utils/osx_bundle/_build/QuodLibet.app/Contents/Resources/lib/gstreamer-1.0/libgstpulseaudio.dylib': dlopen(/Users/jwd/nobackup-cs/quodlibet/dev-utils/osx_bundle/_build/QuodLibet.app/Contents/Resources/lib/gstreamer-1.0/libgstpulseaudio.dylib, 0x0002): Library not loaded: @executable_path/../Resources/lib/pulseaudio/libpulsecommon-16.1.dylib
  Referenced from: <DFE9BF24-7B1C-3ACD-B98B-5BC1C394A881> /Volumes/nobackup-cs/quodlibet/dev-utils/osx_bundle/_build/QuodLibet.app/Contents/Resources/lib/libpulse.0.dylib
  Reason: tried: '/Users/jwd/nobackup-cs/quodlibet/dev-utils/osx_bundle/_build/QuodLibet.app/Contents/Resources/lib/libpulsecommon-16.1.dylib' (no such file), '/Volumes/nobackup-cs/quodlibet/dev-utils/osx_bundle/_build/QuodLibet.app/Contents/Resources/libexec/Resources/lib/pulseaudio/libpulsecommon-16.1.dylib' (no such file)
```

Pulse Audio has its own directory "pulseaudio" within prefix/lib, and libpulsecommon-16.1.dylib is within that.  That private directory isn't included in the listed search paths, so the library isn't found.

The effects of not loading  this plugin are unclear.

**Comment by jdykstra** [comment]
The problem here is the pulseaudio/ private library directory.  gst-plugin-scanner 's real binary (not its symbolic link) is in libexec, and that's where its rpath entry is looking, but the library is actually in lib/.

This wasn't a problem in the previous MacOS release (4.4.0), because the pulseaudio plugin wasn't included in GStreamer.  It's very unclear to me whether this plugin adds any value on the Mac platform.

**Comment by jdykstra** [comment]
Closed with 9493aaf5.
